c2corg_api/models/mailinglist.py

```diff
@@ -0,0 +1,35 @@
+from sqlalchemy import Column, DateTime, ForeignKey, Integer, String
+from sqlalchemy.orm import relationship
+from sqlalchemy.sql.functions import func
+
+from c2corg_api.models import Base, sympa_schema, users_schema
+from c2corg_api.models.user import User
+
+
+class Mailinglist(Base):
+    """
+    Class containing mailing list subscriptions.
+    Based upon Sympa mailing list server https://www.sympa.org/
+    """
+    __tablename__ = 'subscriber_table'
+    __table_args__ = {"schema": sympa_schema}
+
+    listname = Column('list_subscriber', String(50), primary_key=True)
+    email = Column('user_subscriber', String(200), primary_key=True)
+    user_id = Column(
```

Comment:
> There's also a "reverse" index (user_subscriber, list_subscriber). Not created in v6. I don't know if it is really needed.

It depends on how Sympa queries this table. The original sympa database uses an index on `user_subscriber`:
https://github.com/softwerkskammer/server-sympa/blob/dad6a483420ed87583999848f7db055c4059713b/src/etc/script/create_db.Pg#L200-L226

It would make sense to do the same.